content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -37,20 +37,23 @@ The following table presents the list of collected features and the minimal Agen
 |---|---|---|
 | Container Cost Allocation | 7.27.0 | 1.11.0 |
 | GPU Container Cost Allocation | 7.54.0 | 7.54.0 |
-| AWS Persistent Volume Allocation | 7.46.0 | 1.11.0  |
+| AWS Persistent Volume Allocation | 7.46.0 | 1.11.0 |
+| Data Transfer Cost Allocation    | 7.46.0 | 1.11.0 |
```

Comment:
@jeannei Did the npm team get back to you about the minimal agent version and cluster agent version? If not, do you think it's a good idea for me to put the latest current versions here to avoid blocking the PR on waiting for their response?

Reply:
it's `7.55.0`, per conversations we had with the npm team